lib/src/vision/barcode_scanner.dart

```diff
@@ -185,6 +185,11 @@ class BarcodeValue {
   ///
   final BarcodeType type;
 
+  /// The format (symbology) of the barcode value.
+  ///
+  /// For example, [BarcodeFormat.upca], [BarcodeFormat.code128], [BarcodeFormat.dataMatrix]
+  final int format;
```

Comment:
Change this to an enum

Reply:
👍 it does work much nicer as an enum. updated in ad1e5c4